app/admin.py

```diff
@@ -130,6 +128,8 @@ def post(self):
                     ],
                     # These settings are literal strings (not JSON).
                     literal_config_names=[
+                        'unreviewed_notes_email',
+                        'unreviewed_notes_email_threshold',
```

Comment:
As I wrote before, the type of this field would be string if you put this here (in literal_config_names). I believe you need to put this to json_config_names instead to make its type integer, not string.